slack-base/src/main/java/com/hubspot/slack/client/methods/params/reactions/ReactionsAddParamsIF.java

```diff
@@ -17,7 +17,21 @@
   String getName();
   Optional<String> getChannel();
   Optional<String> getTimestamp();
+  /**
+   * @deprecated As of 22nd August 2019, Slack has removed support for file comments -
+   * <a href="https://api.slack.com/changelog/2018-05-file-threads-soon-tread#whats_changed">
+   *   file threads are the replacement
+   * </a>
+   */
+  @Deprecated
```

Comment:
The deprecation comment is identical to the file comments one. Should this not read `As of 22nd August 2019, Slack has removed support for files` ?

Reply:
The `file` field here was related to rendering file comments in the thread.  Slack aren't removing support for files, just file comments.  That's why the javadoc for both these fields reads the same.